docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+slackbot:
+  image: node:6
+  entrypoint: /app/entrypoint.sh
+  environment:
+    - DATABASE_URL=postgres://postgres:standup@database/standup
+    - PORT=3000
+    - LOG_LEVEL=10
+  expose:
+    - 3000
```

Comment:
`expose` is for making container ports available to linked containers.  Since this container doesn't need to be accessible to other containers, I think we can leave out the `expose` line.